### PR TITLE
Created a footer

### LINKED ISSUE #14: Footer Component
image

### PR DESCRIPTION
**Pull request by wx02shi**
closing #14 
image

made a simpler footer compared to the Figma, reasons:
- figma has a small text section, but I literally cannot think of anything productive to put there, so I removed it
- figma has contact info section, but the original site doesn't provide contact info anywhere. I assume all contact is through forms, so I also removed the contact section

added a back to top link

Also corrected the text colour on the landing banner

### CODE REVIEW

**Comment by functionstackx** [comment]
@LemonFace0309 are the assumptions correct or does she want contact at footer?

**Review by functionstackx — COMMENTED** [review]
looks good! Charles can probably approve once he confirms the requirements & the issues below have been resolved

File: components/Footer.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
+import React from 'react'
```
Comment (functionstackx) [inline components/Footer.js:1]: can you own this piece of code

example of how to add to top of file:

image

**Comment by LemonFace0309** [comment]
Should be okay; we can always change later on

**Comment by LemonFace0309** [comment]
Just add author and i'll approve after

**Review by LemonFace0309 — APPROVED** [review]
Nice

**Pull request merged**